# Generated by Django 5.0.1 on 2024-03-28 13:28

import django.contrib.auth.models
import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('request', '0043_alter_agent_options_agent_date_joined_agent_dob_and_more'),
    ]

    operations = [
        migrations.AlterModelManagers(
            name='agent',
            managers=[
                ('objects', django.contrib.auth.models.UserManager()),
            ],
        ),
        migrations.RemoveField(
            model_name='service',
            name='dispursement',
        ),
        migrations.AddField(
            model_name='service',
            name='disbursement',
            field=models.PositiveIntegerField(default=0, verbose_name='Disbursement fee of the service'),
        ),
        migrations.AlterField(
            model_name='agent',
            name='id',
            field=models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID'),
        ),
        migrations.AlterField(
            model_name='agent',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
